sys/hacl/build.rs

```diff
@@ -199,17 +288,23 @@ fn main() {
     eprintln!(" >>> out {:?}", out_path);
 
     // Build the C files
-    build(out_path);
+    build(platform, out_path);
 
     // Set library name to look up
     let library_name = "hacl";
```

Comment:
Let's move this out into a `const` together with the other library names for 128 and 256 etc.

Reply:
Edited!

Reply:
I switched `library_name` to `const`, let me know if I missed something here.